mm2src/coins/lightning.rs

```diff
@@ -387,6 +390,43 @@ impl MarketCoinOps for LightningCoin {
 
     fn my_address(&self) -> Result<String, String> { Ok(self.my_node_id()) }
 
+    fn get_public_key(&self) -> Result<String, MmError<UnexpectedDerivationMethod>> { unimplemented!() }
+
+    fn sign_message_hash(&self, message: &str) -> Option<[u8; 32]> {
+        let mut _message_prefix = self.conf.sign_message_prefix.clone()?;
+        let prefixed_message = format!("{}{}", _message_prefix, message);
+        Some(dhash256(prefixed_message.as_bytes()).take())
+    }
+
+    fn sign_message(&self, message: &str) -> SignatureResult<String> {
+        let message_hash = self.sign_message_hash(message).ok_or(SignatureError::PrefixNotFound)?;
+        let secret_key = self
+            .keys_manager
+            .get_node_secret(Recipient::Node)
+            .map_err(|_| SignatureError::InternalError("Error accessing node keys".to_string()))?;
+        let private = Private {
+            prefix: 239,
+            secret: H256::from_str(&secret_key.to_string())
```

Comment:
`H256::from(*secret_key.as_ref())` doesn't require `to_string` conversion and possible error handling.